javalib/src/main/scala/java/time/Year.scala

```diff
@@ -0,0 +1,203 @@
+package java.time
+
+import java.time.chrono.{Chronology, IsoChronology}
+import java.time.temporal._
+
+/** Created by alonsodomin on 24/12/2015. */
+final class Year private (year: Int)
+    extends TemporalAccessor with Temporal with TemporalAdjuster with Comparable[Year]
+    with java.io.Serializable {
+
+  import Preconditions._
+  import Year._
+  import ChronoField._
+  import ChronoUnit._
+
+  requireDateTime(year >= MIN_VALUE && year <= MAX_VALUE, s"Invalid year value: $year")
+
+  def getValue(): Int = year
+
+  def isSupported(field: TemporalField): Boolean = field match {
+    case _: ChronoField => field == YEAR_OF_ERA || field == YEAR || field == ERA
+    case null           => false
+    case _              => field.isSupportedBy(this)
+  }
+
+  def isSupported(unit: TemporalUnit): Boolean = unit match {
+    case _: ChronoUnit => unit == YEARS || unit == DECADES || unit == CENTURIES ||
+      unit == MILLENNIA || unit == ERAS
```

Comment:
Start `unit == YEARS || ...` on the next line and add a new line before the next `case`.